diff --git a/services/Diten.Platform/Diten.Platform.sln b/services/Diten.Platform/Diten.Platform.sln
index 90933a1f4..889c39fd3 100644
--- a/services/Diten.Platform/Diten.Platform.sln
+++ b/services/Diten.Platform/Diten.Platform.sln
@@ -17,6 +17,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{238E8832
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Diten.Platform.Application.Tests", "tests\Diten.Platform.Application.Tests\Diten.Platform.Application.Tests.csproj", "{81CAF06E-0BC1-4D7D-A7B2-37C2638AB073}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Diten.Platform.CompositionGuard.Tests", "tests\Diten.Platform.CompositionGuard.Tests\Diten.Platform.CompositionGuard.Tests.csproj", "{2A146FD7-7D9E-42A9-9A97-BD6BA48F26FF}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
@@ -46,6 +48,10 @@ Global
 		{81CAF06E-0BC1-4D7D-A7B2-37C2638AB073}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{81CAF06E-0BC1-4D7D-A7B2-37C2638AB073}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{81CAF06E-0BC1-4D7D-A7B2-37C2638AB073}.Release|Any CPU.Build.0 = Release|Any CPU
+		{2A146FD7-7D9E-42A9-9A97-BD6BA48F26FF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{2A146FD7-7D9E-42A9-9A97-BD6BA48F26FF}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{2A146FD7-7D9E-42A9-9A97-BD6BA48F26FF}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{2A146FD7-7D9E-42A9-9A97-BD6BA48F26FF}.Release|Any CPU.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
 		{085A5523-6759-4326-9AFC-A6B1DE110C64} = {AE172E6A-2B37-4738-95BF-787B366FC6E7}
@@ -53,5 +59,6 @@ Global
 		{2EB74C81-6D5E-4205-9ACA-D16657CE1A6F} = {AE172E6A-2B37-4738-95BF-787B366FC6E7}
 		{DD3E5971-EC5D-40A5-A658-BF5B7DABC0AF} = {AE172E6A-2B37-4738-95BF-787B366FC6E7}
 		{81CAF06E-0BC1-4D7D-A7B2-37C2638AB073} = {238E8832-E494-4DA1-A554-44A3462CDD69}
+		{2A146FD7-7D9E-42A9-9A97-BD6BA48F26FF} = {238E8832-E494-4DA1-A554-44A3462CDD69}
 	EndGlobalSection
 EndGlobal
